week-4-practise/level-1/exercise-1.js

```diff
@@ -36,9 +36,45 @@ var writers = [
     occupation: "writer",
     age: 64,
     alive: true
-  },
+  }
 ];
+var livingWriters = [];
+
+var writer = "writer";
+
+function isAlive(writer) {
+  return writers
+    .filter(
+      authour => authour.occupation === "writer" && authour.alive === true
+    )
+    .map(writers => writers);
+}
+livingWriters = isAlive(writer);
 
+console.log(
+  "Hi my name is " +
+    livingWriters[0].firstName +
+    " " +
+    livingWriters[0].lastName +
+    ". " +
+    "I am " +
+    livingWriters[0].age +
+    " years old, and work as a " +
+    livingWriters[0].occupation +
+    "."
+);
+console.log(

```

Comment:
You're repeating a lot of code in these two calls of `console.log()`. Image you had 20 more writers that were alive – you'd have to add so much code! Can you think of a way to make this more DRY (don't repeat yourself)? Maybe use a function.